ResourceStatusSystem/PolicySystem/Actions/FTSStorageUnbanAction.py

```diff
@@ -0,0 +1,82 @@
+# $HeadURL:  $
+''' FTSstorageUnbanAction
+
+'''
+
+from DIRAC                                                      import gConfig, gLogger, S_ERROR, S_OK
+from DIRAC.Interfaces.API.DiracAdmin                            import DiracAdmin
+from DIRAC.ResourceStatusSystem.PolicySystem.Actions.BaseAction import BaseAction
+from DIRAC.ResourceStatusSystem.Utilities                       import RssConfiguration
+#from DIRAC.ResourceStatusSystem.Utilities.InfoGetter            import InfoGetter
+from DIRAC.Core.Security.ProxyInfo                              import getProxyInfo
+from DIRAC.ConfigurationSystem.Client.Helpers.Resources         import getFTSServers
+import fts3.rest.client.easy as fts3
+import json
+
+from DIRAC import gLogger
+
+__RCSID__  = '$Id:  $'
+
+class FTSStorageUnbanAction( BaseAction ):
+	'''
+		Action that sends to the FTS server a unbanning request for a given list of sites.
+	'''
+	
+	def __init__( self, name, decissionParams, enforcementResult, singlePolicyResults, clients ):
+		
+		super( FTSStorageUnbanAction, self ).__init__( name, decissionParams, enforcementResult, 
+                                         singlePolicyResults, clients )
+		self.diracAdmin = DiracAdmin()
```

Comment:
As far as I understand, no I don't need it. I had it in my code b/c I took the EmailAction as a template. Ok I'll remove it.